src/framework/Framework_Headers_Cache_002.test.js

```diff
@@ -0,0 +1,79 @@
+import {
+  getPluginInfo,
+  getResponseForURLRequest,
+  restartFramework,
+} from '../commonMethods/commonFunctions'
+import constants from '../commonMethods/constants'
+
+let count = 0
+export default {
+  title: 'Validate cache headers on all plugins',
```

Comment:
This test validates the REST APIs of the Thunder stack. REST APIs are deprecated and I think we should not be testing for them anymore. Thoughts @Divya-563 ?

Reply:
Hello @Divya-563 , @wouterlucas ,
Please let me know whether this test case can be dropped .

Reply:
Yes this can be dropped.